packages/plugins/rubygem-smart_proxy_dynflow/rubygem-smart_proxy_dynflow.spec

```diff
@@ -18,9 +18,11 @@ Requires: ruby(rubygems)
 Requires: foreman-proxy >= 1.12.0
 
 %if 0%{?fedora}
-Requires: rubygem(smart_proxy_dynflow_core) = %{version}
+Requires: rubygem(smart_proxy_dynflow_core) >= 0.2.0
```

Comment:
Due to urgency of this release and the fact that smart_proxy_dynflow_core build was currently failling (http://ci.theforeman.org/blue/organizations/jenkins/foreman-packaging-rpm-pr-test/detail/foreman-packaging-rpm-pr-test/1658/pipeline/), I'm releasing the requirement on the core package here, as there is nothing we need to rely on from the core package right now, that would require the strict version constraint